pkg/kubectl/cmd/util/printing.go

```diff
@@ -30,6 +34,40 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
 }
 
+// AddOutputFlagsForMutation adds output related flags to a command. Used by mutations only.
+func AddOutputFlagsForMutation(cmd *cobra.Command) {
+	cmd.Flags().StringP("output", "o", "", "Output mode. Use \"-o name\" for shorter output (resource/name).")
```

Comment:
Optional: helper method to inspect the flag for the expected value.  It seems a little strange that commands call this to register a flag, but there's not a corresponding test for flag in this package.  I don't feel strongly though.